src/PageContainer/component.js

```diff
@@ -100,8 +100,10 @@ export default class PageContainer extends Component {
     }
 
     const item = find(context.collection, pageUrl)
-
+    console.log(isClient)
+    console.log(typeof window !== "undefined")
```

Comment:
Looks like a bad design to need this thing.
What about finding a way to pass this as a prop?